### ISSUE TITLE
xml.etree.ElementTree.ParseError: not well-formed (invalid token)

### ISSUE THREAD

**Issue by Sun-Ke** [thanhnghiadk/syntactic_HME_generation#2]
I got this error when I tried to parse the output inkml files using `xml.etree.ElementTree.parse`:
```
xml.etree.ElementTree.ParseError: not well-formed (invalid token)
```
From my observation, I think the less than symbol and the greater than symbol in ground truth make this happen because they are **escape characters** in xml. For example, "`<annotation type="truth"> a < b </annotation>`" would totally ruin xml format and cause parse error when converting inkml files to images.

Maybe we should replace '<' with '\lt' and replace '>' with '\gt' before output? (in `inkml.py`)

Not that it matters, but it frustrates me that the output data is unavailable after long time running...:joy:  Sorry for my bad English :)

**Comment by thanhnghiadk** [comment]
Yes, thank you for your advice. We should replace '<' with '\lt' and replace '>' with '\gt.
